Python/bytecodes.c

```diff
@@ -1049,7 +1049,7 @@ dummy_func(
             assert(frame != &entry_frame);
             frame->instr_ptr = next_instr;
             PyGenObject *gen = _PyFrame_GetGenerator(frame);
-            gen->gi_frame_state = FRAME_SUSPENDED;
+            gen->gi_frame_state = oparg ? FRAME_SUSPENDED_YIELD_FROM : FRAME_SUSPENDED;
```

Comment:
Can we make this unconditional?
Ideally `gen->gi_frame_state = oparg`, but that might require flipping the sign of the all the gen state constants.
`gen->gi_frame_state = oparg + K` would be OK.